lib/CleantalkSP/SpbctWP/Scanner/ScannerQueue.php

```diff
@@ -1582,22 +1596,45 @@ public function send_results(){
                 $unknown[$path] = array_values($row);
             }
         }
-        
-        // Count files to scan
-        $scanned_total = $this->countFileSystem();
-        $scanned_total = $scanned_total['total'];
-        $error         = '';
-        
+
+        $error = '';
+
+        /**
+         * Getting parameters for security_mscan_logs()
+         */
+        $key = $spbc->settings['spbc_key'];
+        $unknown = $spbc->settings['scanner__list_unknown'];
+        $service_id = $spbc->service_id;
+        $scan_date = current_time('Y-m-d H:i:s');
+        $scan_result = $modified ? 'warning' : 'passed';
+        $total_site_files = $spbc->data['scanner']['files_total'] = $this->countFileSystem()['total'];
+        $failed_files = $modified;
+        $unknown_files = $unknown;
+        $scan_type = RemoteCalls::check() ? 'auto': 'manual';
+        $checksums_count_ct = isset($spbc->data['scanner']['checksums_count_ct']) ? $spbc->data['scanner']['checksums_count_ct'] : null;
+        $checksums_count_user = (int) $wpdb->get_var('SELECT COUNT(*) from ' . SPBC_TBL_SCAN_FILES. ' WHERE status = "APROVED"');
```

Comment:
may be to use function for example getCountScannedAprovedFiles()

Reply:
Maybe in the future, but I do not to be shure where we can place such function/method.